lib/dor/services/client/object_version.rb

```diff
@@ -52,7 +50,10 @@ def openable?(**params)
         # rubocop:enable Metrics/MethodLength
 
         # Open new version for an object
-        # @param params [Hash] optional params (see dor-services-app)
+        # @param :description [String] (required) - a description of the object version being opened
```

Comment:
please remove the leading colons here. That's not valid yard doc. Additionally yard specifies that the default value goes in the parenthesis, so we should not put `required` there.  All arguments are required unless they have a default.

Reply:
ok.